teamscale-maven-plugin/pom.xml

```diff
@@ -54,7 +54,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <revision>1.0.0-SNAPSHOT</revision>
-        <teamscale.agent.version>32.4.2</teamscale.agent.version>
+        <teamscale.agent.version>29.0.0</teamscale.agent.version>
```

Comment:
Does this need to be 29.0.0? If so I assume the change might have originated from renovate and we should disable it for this file and this dependency to no longer cause an automated update of it

Reply:
I don't think that's renovate. Something in the build process is overwriting this value in this file whenever I run tests locally. Haven't figured out yet what that is. Sorry for accidentally checking this in

Reply:
💡 Ah this is https://github.com/cqse/teamscale-jacoco-agent/blob/master/teamscale-maven-plugin/build.gradle.kts#L12

Reply:
Maybe a comment helps to avoid this confusion